net-libs/neon/neon-0.32.2.ebuild

```diff
@@ -25,30 +26,17 @@ RDEPEND="expat? ( dev-libs/expat:0=[${MULTILIB_USEDEP}] )
 			app-misc/ca-certificates
 			net-libs/gnutls:0=[${MULTILIB_USEDEP}]
 		)
-		!gnutls? (
-			dev-libs/openssl:0=[${MULTILIB_USEDEP}]
-		)
+		!gnutls? ( dev-libs/openssl:0=[${MULTILIB_USEDEP}] )
 		pkcs11? ( dev-libs/pakchois:0=[${MULTILIB_USEDEP}] )
 	)
 	zlib? ( sys-libs/zlib:0=[${MULTILIB_USEDEP}] )"
-DEPEND="${RDEPEND}"
-BDEPEND="
-	app-text/docbook-xml-dtd:4.5
-	app-text/xmlto
-	virtual/pkgconfig
-"
+RDEPEND="${DEPEND}"
 
 MULTILIB_CHOST_TOOLS=(
 	/usr/bin/neon-config
 )
 
 src_prepare() {
-	# Use CHOST-prefixed version of xml2-config for cross-compilation.
-	sed -e "s/AC_CHECK_PROG(XML2_CONFIG,/AC_CHECK_TOOL(XML2_CONFIG,/" -i macros/neon-xml-parser.m4 || die "sed failed"
-
-	# Fix compatibility with OpenSSL >=1.1.
-	sed -e "s/RSA_F_RSA_PRIVATE_ENCRYPT/RSA_F_RSA_OSSL_PRIVATE_ENCRYPT/" -i src/ne_pkcs11.c || die "sed failed"
-
 	eapply_user
 
 	AT_M4DIR="macros" eautoreconf
```

Comment:
For the code below:
- replace `dodoc -r doc/html/*` with `dodoc -r doc/html/.`
- replace ${D} with ${ED} in `find "${D}" -name "*.la" ...`
- replace `dodoc` line with `einstalldocs` (since all of those are handled by default with [einstalldocs](https://devmanual.gentoo.org/ebuild-writing/functions/src_install/index.html#default-src_install))

Reply:
Yes for all, but with `einstalldocs`, I will use set `DOCS` explicitly without `ChangeLog`, which is useless:

```
$ cat ChangeLog
------------------------------------------------------------------------
r1734 | joe.orton | 2022-01-11 12:20:48 +0000 (Tue, 11 Jan 2022) | 2 lines

Bump for 0.32.2.

------------------------------------------------------------------------
```